src/quo/components/wallet/account_card/view.cljs

```diff
@@ -96,87 +95,86 @@
-        on-press-in  #(reset! pressed? true)
-        on-press-out #(reset! pressed? false)]
-    (fn [{:keys [name balance percentage-value loading? amount customization-color type emoji metrics?
-                 theme on-press]}]
-      (let [watch-only?      (= :watch-only type)
-            missing-keypair? (= :missing-keypair type)]
-        (if loading?
-          [loading-view
-           {:customization-color customization-color
-            :type                type
-            :theme               theme
-            :metrics?            metrics?}]
-          [rn/pressable
-           {:on-press-in  on-press-in
-            :on-press-out on-press-out
-            :style        (style/card {:customization-color customization-color
-                                       :type                type
-                                       :theme               theme
-                                       :pressed?            @pressed?
-                                       :metrics?            metrics?})
-            :on-press     on-press}
-           (when (and customization-color (and (not watch-only?) (not missing-keypair?)))
-             [customization-colors/overlay
-              {:customization-color customization-color
-               :border-radius       16
-               :theme               theme
-               :pressed?            @pressed?}])
-           [rn/view {:style style/profile-container}
-            [rn/view {:style {:padding-bottom 2 :margin-right 2}}
-             [text/text {:style style/emoji} emoji]]
-            [rn/view {:style style/watch-only-container}
-             [text/text
-              {:size            :paragraph-2
-               :weight          :medium
-               :number-of-lines 1
-               :max-width       110
-               :margin-right    4
-               :ellipis-mode    :tail
-               :style           (style/account-name type theme)}
-              name]
-             (when watch-only? [icon/icon :i/reveal {:color colors/neutral-50 :size 12}])
-             (when missing-keypair?
-               [icon/icon :i/alert {:color (properties/alert-icon-color theme) :size 12}])]]
-           [text/text
-            {:size   :heading-2
-             :weight :semi-bold
-             :style  (style/account-value type theme)}
-            balance]
-           (when metrics?
-             [rn/view {:style style/metrics-container}
-              [metrics-percentage type theme percentage-value]
-              (when (not= :empty type)
-                [metrics-info type theme amount])])
-           (when watch-only?
-             [gradient-overview theme customization-color])])))))
+  [{:keys [name balance percentage-value loading? amount customization-color type emoji metrics?
+           on-press]}]
+  (let [theme                  (quo.theme/use-theme-value)
+        [pressed? set-pressed] (rn/use-state false)
+        on-press-in            (rn/use-callback #(set-pressed true))
```

Comment:
it seems there is some questions on how much memoization we want to add to components. Particularly simple ui components like this which don't really have much need /use for this level of optimzation. As discussed in https://github.com/status-im/status-mobile/pull/19298 by @BalogunofAfrica.
It would be nice to figure this detail out before we merge all these refactor pr's so that we don't have to refactor the refactors :)

Reply:
all should be memoized, it's cheap

Reply:
@flexsurfer It does comes at a cost of more memory been used. I would be curious to see how this play out in a resource constrained (low RAM) Android device

Reply:
And this would also hold true for us memoizing every component by default

Reply:
@flexsurfer I also think memoizing everything is too much.

Memoizing callbacks is important when we pass these callbacks to components that will re-render but not all of our components suffer from it. IMO applying memoization (in general, not just React) is a process done after identifying bottlenecks, not a thing that must be done all the time since it's not for free.

Also @cammellos was in favour of making this rule more leninent:
https://github.com/status-im/status-mobile/pull/19033#discussion_r1509245855

wdyt?

Reply:
reagent already memo hundreds of components by default, adding a few callbacks won't change the picture much, in react it's not done by default because comparison is expensive and not  always possible or effective not because of memory

Reply:
Well, reagent memoizes (makes use of `memo`) because it needs to add a custom way to compare data across rerenders,  otherwise, everything may be re-rendered all the time.

> adding a few callbacks won't change the picture much

They aren't few callbacks, they'd be all the callbacks.

Anyway, again just wanted to share my thoughts on this topic.